### PR TITLE
Link code snippet

### PR DESCRIPTION
**Pull request by nielslange**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

In #39176, @nerrad introduced a code snippets directory with a README.md file, that lists all containing code snippets. In #39177, he then added the first code snippet. This PR adds the link to this code snippet to the README.md file.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Look up the changed README.md.
2. Verify that the link `Add a message above the login / register form` leads to the corresponding code snippet.

### CODE REVIEW

File: docs/snippets/README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # WooCommerce code snippets
 
 Various code snippets you can add to your site to enable various custom functionality.
+
+-   [Add a message above the login / register form](./before-login--register-form.md)
```
Comment (nielslange) [inline docs/snippets/README.md:5]: I was thinking of adding sub headlines to the file, e.g. Frontend Code Snippets (see https://github.com/woocommerce/woodocs/issues/41). Given that it's only one code snippet now and that it's unknown how many code snippets will be added, I decided against it.

Comment (nerrad) [reply docs/snippets/README.md:5]: Yup good call. We can add sub-headings as needed.

**Review by nerrad — APPROVED** [review]

File: docs/snippets/README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
 # WooCommerce code snippets
 
 Various code snippets you can add to your site to enable various custom functionality.
```
Comment (nerrad) [inline docs/snippets/README.md]: Suggest removing the superfluous `various` here:
```suggestion
Various code snippets you can add to your site to enable custom functionality:
```

Comment (nerrad) [reply docs/snippets/README.md]: I pre-approved so when you make the suggested changes you can just go ahead and merge.

**Pull request merged**